dts/bindings/adc/nuvoton,adc-cmp.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright (c) 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    This will perform signal comparision with threshold established.
+
+compatible: "nuvoton,adc-cmp"
```

Comment:
This binding needs to go under `dts/bindings/sensor` as the driver implementing this is a sensor driver.

Reply:
Done.